data/json/overmap_specials.json

```diff
@@ -209,6 +209,18 @@
         "flags" : [ "CLASSIC" ]
     },{
         "type" : "overmap_special",
+        "id" : "Sugar House",
+        "overmaps" : [
+            { "point":[0,0,0], "overmap": "sugar_house", "connect": "road" }
```

Comment:
This format isn't supported anymore. After [the recent changes](https://github.com/CleverRaven/Cataclysm-DDA/pull/19144/commits/88dbe396bc59279d3068ddac7095c2ba4588ab89), `"connect": "road"` can be rewritten like this:
```json
"connections" : [
    { "point" : [0,-1,0], "terrain" : "road" }
],
```